
<!DOCTYPE html>


<html lang="en" data-content_root="./" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Scenario 5: Genetic Analysis Case Study on NRAS gene &#8212; EJP-RD FDP and Virtual Portal demonstrator</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=fa44fd50" />
    <link rel="stylesheet" type="text/css" href="_static/styles/sphinx-book-theme.css?v=a3416100" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-design.min.css?v=87e54e7c" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="_static/doctools.js?v=9a2dae69"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="_static/copybutton.js?v=f281be69"></script>
    <script src="_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'scenario5';</script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="prev" title="Scenario 4: Genetic Analysis on AML" href="scenario4.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="index.html">
  
  
  
  
  
    
    
      
    
    
    <img src="_static/logo.png" class="logo__image only-light" alt="EJP-RD FDP and Virtual Portal demonstrator - Home"/>
    <script>document.write(`<img src="_static/logo.png" class="logo__image only-dark" alt="EJP-RD FDP and Virtual Portal demonstrator - Home"/>`);</script>
  
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="index.html">
                    Welcome to the EJP-RD FDP Infrastructure and VP Portal Demonstrator
                </a>
            </li>
        </ul>
        <ul class="current nav bd-sidenav">
<li class="toctree-l1"><a class="reference internal" href="scenario1.html">Scenario 1: Genetic Analysis Case Study on FBX011 gene.</a></li>
<li class="toctree-l1"><a class="reference internal" href="scenario2.html">Scenario 2: Genetic Analysis Case Study on MAPK1 gene</a></li>
<li class="toctree-l1"><a class="reference internal" href="scenario3.html">Scenario 3: Genetic Analysis Case Study on FN1 gene</a></li>
<li class="toctree-l1"><a class="reference internal" href="scenario4.html">Scenario 4: Genetic Analysis on AML</a></li>
<li class="toctree-l1 current active"><a class="current reference internal" href="#">Scenario 5: Genetic Analysis Case Study on NRAS gene</a></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/ejp-rd-vp/DistributedAnalysisDemonstrator" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/ejp-rd-vp/DistributedAnalysisDemonstrator/issues/new?title=Issue%20on%20page%20%2Fscenario5.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="_sources/scenario5.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Scenario 5: Genetic Analysis Case Study on NRAS gene</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#an-undiagnosed-child-with-delayed-motor-development-and-generalized-dysmorphism">An Undiagnosed Child with Delayed Motor Development and Generalized Dysmorphism</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-questions">Research Questions</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#question-1-gene-mutation-prevalence">Question 1: Gene Mutation Prevalence</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#question-2-phenotype-and-disease-prevalence">Question 2: Phenotype and Disease Prevalence</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#question-3-matching-phenotypes">Question 3: Matching Phenotypes</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#conclusion">Conclusion</a></li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="scenario-5-genetic-analysis-case-study-on-nras-gene">
<h1>Scenario 5: Genetic Analysis Case Study on NRAS gene<a class="headerlink" href="#scenario-5-genetic-analysis-case-study-on-nras-gene" title="Link to this heading">#</a></h1>
<section id="an-undiagnosed-child-with-delayed-motor-development-and-generalized-dysmorphism">
<h2>An Undiagnosed Child with Delayed Motor Development and Generalized Dysmorphism<a class="headerlink" href="#an-undiagnosed-child-with-delayed-motor-development-and-generalized-dysmorphism" title="Link to this heading">#</a></h2>
<p>On clinical examination, the undiagnosed child was found to have several facial dysmorphisms, in particular a broad neck and low-set ears. It has also been assessed that the child exhibits delayed motor development.</p>
<p>Genetic testing has been performed using Whole Exome Sequencing (WES) which identified a probable pathogenic mutation in the NRAS gene.</p>
</section>
<hr class="docutils" />
<section id="research-questions">
<h2>Research Questions<a class="headerlink" href="#research-questions" title="Link to this heading">#</a></h2>
<section id="question-1-gene-mutation-prevalence">
<h3>Question 1: Gene Mutation Prevalence<a class="headerlink" href="#question-1-gene-mutation-prevalence" title="Link to this heading">#</a></h3>
<p>Are there any other individuals with a mutation in the same gene?</p>
<p>To answer this question the SPARQL query below has been used for finding the phenopackets that include at least one mutation in the NRAS gene. To identify the correct gene descriptions, the HGNC gene ID is used of NRAS being <a class="reference external" href="https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/7989">HGNC:7989</a>.</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="s2">&quot;PREFIX ex: &lt;https://example.org/&gt;</span><span class="se">\n</span><span class="s2">PREFIX obo: &lt;http://purl.obolibrary.org/obo/&gt;</span><span class="se">\n</span><span class="s2">PREFIX sio: &lt;http://semanticscience.org/resource/&gt;</span><span class="se">\n</span><span class="s2">PREFIX dcterms: &lt;http://purl.org/dc/terms/&gt; </span><span class="se">\n</span><span class="s2">PREFIX rdfs: &lt;http://www.w3.org/2000/01/rdf-schema#&gt;</span><span class="se">\n\n</span><span class="s2">SELECT ?phenopacketidval ?creatorname ?createddate</span><span class="se">\n</span><span class="s2">WHERE {</span><span class="se">\n</span><span class="s2">    # Find the phenopackets that contain at least one mutation in given gene</span><span class="se">\n</span><span class="s2">    ?genedescr a obo:NCIT_C16612 ;</span><span class="se">\n</span><span class="s2">               dcterms:identifier &#39;HGNC:7989&#39; .</span><span class="se">\n</span><span class="s2">    ?vardescr sio:SIO_001403 ?genedescr .</span><span class="se">\n</span><span class="s2">    ?varinterpr sio:SIO_001403 ?vardescr .</span><span class="se">\n</span><span class="s2">    ?genomicinterp sio:SIO_001403 ?varinterpr .</span><span class="se">\n</span><span class="s2">    ?diagnosis sio:SIO_001403 ?genomicinterp .</span><span class="se">\n</span><span class="s2">    ?interpr sio:SIO_001403 ?diagnosis .</span><span class="se">\n</span><span class="s2">    ?phenopacket sio:SIO_001403 ?interpr ;</span><span class="se">\n</span><span class="s2">                 a obo:NCIT_C79269 ;</span><span class="se">\n</span><span class="s2">                 sio:SIO_000228 ?role .</span><span class="se">\n</span><span class="s2">    </span><span class="se">\n</span><span class="s2">    # Find phenopacket ID value</span><span class="se">\n</span><span class="s2">    ?phenopacketid sio:SIO_000020 ?role ;</span><span class="se">\n\t</span><span class="s2">               sio:SIO_000300 ?phenopacketidval .</span><span class="se">\n</span><span class="s2">    </span><span class="se">\n</span><span class="s2">    # Find metadata</span><span class="se">\n</span><span class="s2">    ?phenopacket sio:SIO_001403 ?metadata .</span><span class="se">\n</span><span class="s2">    ?metadata a obo:NCIT_C52095 ;</span><span class="se">\n</span><span class="s2">              sio:SIO_000008 ?creator ;</span><span class="se">\n</span><span class="s2">              sio:SIO_000008 ?created .</span><span class="se">\n</span><span class="s2">    </span><span class="se">\n</span><span class="s2">    ?creator a dcterms:creator .</span><span class="se">\n</span><span class="s2">    ?creatorval sio:SIO_000628 ?creator ;</span><span class="se">\n</span><span class="s2">                a obo:NCIT_C70856 ;</span><span class="se">\n</span><span class="s2">                sio:SIO_000300 ?creatorname .</span><span class="se">\n</span><span class="s2">    </span><span class="se">\n</span><span class="s2">    ?created a dcterms:created .</span><span class="se">\n</span><span class="s2">    ?createdval sio:SIO_000628 ?created ;</span><span class="se">\n</span><span class="s2">                a obo:NCIT_C70856 ;</span><span class="se">\n</span><span class="s2">                sio:SIO_000300 ?createddate .</span><span class="se">\n</span><span class="s2">}&quot;</span>
</pre></div>
</div>
<p>As shown below, a total of fourteen phenopackets have been found including phenotypic and genetic information related to mutations in the NRAS gene. This set of phenopackets has been created by the same researcher at the same datetime based on the metadata.</p>
<div class="output text_html"><div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>phenopacket id</th>
      <th>creator</th>
      <th>creation date</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>PMID_26467218_individual_6_Cirstea_et_al__14</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>1</th>
      <td>PMID_26467218_individual_11_Kraoua_et_al__23</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>2</th>
      <td>PMID_26467218_individual_3_Denayer_et_al__22</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>3</th>
      <td>PMID_26467218_individual_9_Cirstea_et_al__14</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>4</th>
      <td>PMID_26467218_individual_4_Denayer_et_al__22</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>5</th>
      <td>PMID_26467218_individual_12_Present_study</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>6</th>
      <td>PMID_26467218_individual_1_De_Filippi_et_al__20</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>7</th>
      <td>PMID_26467218_individual_2_Runtuwene_et_al__21</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>8</th>
      <td>PMID_26467218_individual_9_M_Cirstea_et_al__14</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>9</th>
      <td>PMID_26467218_individual_5_Denayer_et_al__22</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>10</th>
      <td>PMID_26467218_individual_12_F_Present_study</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>11</th>
      <td>PMID_26467218_individual_8_Cirstea_et_al__14</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>12</th>
      <td>PMID_26467218_individual_10_Kraoua_et_al__23</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
    <tr>
      <th>13</th>
      <td>PMID_26467218_individual_7_Cirstea_et_al__14</td>
      <td>ORCID:0000-0002-0736-9199</td>
      <td>2024-05-20 20:56:12</td>
    </tr>
  </tbody>
</table>
</div></div></section>
<section id="question-2-phenotype-and-disease-prevalence">
<h3>Question 2: Phenotype and Disease Prevalence<a class="headerlink" href="#question-2-phenotype-and-disease-prevalence" title="Link to this heading">#</a></h3>
<p>What phenotypes and which diseases have been observed in these individuals and how often do they occur in this group of individuals?</p>
<p>The phenopackets found for the previous question can now be analysed to conclude which phenotypes and diseases occur in this group of individuals. This is done by using SPARQL queries for acquiring the associated phenotypes and diseases. The query below searches for all diagnosed diseases in the relevant phenopackets.</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="s1">&#39;PREFIX ex: &lt;https://example.org/&gt;</span><span class="se">\n</span><span class="s1">PREFIX obo: &lt;http://purl.obolibrary.org/obo/&gt;</span><span class="se">\n</span><span class="s1">PREFIX sio: &lt;http://semanticscience.org/resource/&gt;</span><span class="se">\n</span><span class="s1">PREFIX dcterms: &lt;http://purl.org/dc/terms/&gt; </span><span class="se">\n</span><span class="s1">PREFIX rdfs: &lt;http://www.w3.org/2000/01/rdf-schema#&gt;</span><span class="se">\n\n</span><span class="s1">SELECT ?idvalue ?diseaseid ?diseaselabel</span><span class="se">\n</span><span class="s1">WHERE {</span><span class="se">\n</span><span class="s1">    # Find relevant phenopackets</span><span class="se">\n</span><span class="s1">    ?id a obo:IAO_0020000 ;</span><span class="se">\n</span><span class="s1">        sio:SIO_000300 ?idvalue .</span><span class="se">\n</span><span class="s1">    VALUES ?idvalue { &quot;PMID_26467218_individual_6_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_11_Kraoua_et_al__23&quot; &quot;PMID_26467218_individual_3_Denayer_et_al__22&quot; &quot;PMID_26467218_individual_9_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_4_Denayer_et_al__22&quot; &quot;PMID_26467218_individual_12_Present_study&quot; &quot;PMID_26467218_individual_1_De_Filippi_et_al__20&quot; &quot;PMID_26467218_individual_2_Runtuwene_et_al__21&quot; &quot;PMID_26467218_individual_9_M_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_5_Denayer_et_al__22&quot; &quot;PMID_26467218_individual_12_F_Present_study&quot; &quot;PMID_26467218_individual_8_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_10_Kraoua_et_al__23&quot; &quot;PMID_26467218_individual_7_Cirstea_et_al__14&quot; } .</span><span class="se">\n</span><span class="s1">    </span><span class="se">\n</span><span class="s1">    # Find diagnosis</span><span class="se">\n</span><span class="s1">    ?id sio:SIO_000020 ?role .</span><span class="se">\n</span><span class="s1">    ?phenopacket sio:SIO_000228 ?role ;</span><span class="se">\n</span><span class="s1">                 sio:SIO_001403 ?interpr .</span><span class="se">\n</span><span class="s1">    ?interpr a obo:NCIT_C41255 ;</span><span class="se">\n</span><span class="s1">             sio:SIO_001403 ?diagnosis .</span><span class="se">\n</span><span class="s1">    ?diagnosis a sio:SIO_000614 ;</span><span class="se">\n</span><span class="s1">               sio:SIO_001403 ?disease .</span><span class="se">\n</span><span class="s1">    ?diseaseval a obo:NCIT_C2991 ;</span><span class="se">\n</span><span class="s1">                sio:SIO_000628 ?disease .</span><span class="se">\n</span><span class="s1">    ?disease rdfs:label ?diseaselabel ;</span><span class="se">\n</span><span class="s1">             dcterms:identifier ?diseaseid .</span><span class="se">\n</span><span class="s1">}&#39;</span>
</pre></div>
</div>
<p>Given the results of the SPARQL query shown above, the individuals for which mutations are found in the NRAS gene are all diagnosed with Noonan syndrome 6.</p>
<div class="output text_html"><div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>phenopacket id</th>
      <th>disease ID</th>
      <th>disease label</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>PMID_26467218_individual_6_Cirstea_et_al__14</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>1</th>
      <td>PMID_26467218_individual_11_Kraoua_et_al__23</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>2</th>
      <td>PMID_26467218_individual_3_Denayer_et_al__22</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>3</th>
      <td>PMID_26467218_individual_9_Cirstea_et_al__14</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>4</th>
      <td>PMID_26467218_individual_4_Denayer_et_al__22</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>5</th>
      <td>PMID_26467218_individual_12_Present_study</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>6</th>
      <td>PMID_26467218_individual_1_De_Filippi_et_al__20</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>7</th>
      <td>PMID_26467218_individual_2_Runtuwene_et_al__21</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>8</th>
      <td>PMID_26467218_individual_9_M_Cirstea_et_al__14</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>9</th>
      <td>PMID_26467218_individual_5_Denayer_et_al__22</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>10</th>
      <td>PMID_26467218_individual_12_F_Present_study</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>11</th>
      <td>PMID_26467218_individual_8_Cirstea_et_al__14</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>12</th>
      <td>PMID_26467218_individual_10_Kraoua_et_al__23</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
    <tr>
      <th>13</th>
      <td>PMID_26467218_individual_7_Cirstea_et_al__14</td>
      <td>OMIM:613224</td>
      <td>Noonan syndrome 6</td>
    </tr>
  </tbody>
</table>
</div></div><p>The query shown below searches for all phenotypes that are associated with at least one of the relevant phenopackets.</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="s1">&#39;PREFIX ex: &lt;https://example.org/&gt;</span><span class="se">\n</span><span class="s1">PREFIX obo: &lt;http://purl.obolibrary.org/obo/&gt;</span><span class="se">\n</span><span class="s1">PREFIX sio: &lt;http://semanticscience.org/resource/&gt;</span><span class="se">\n</span><span class="s1">PREFIX dcterms: &lt;http://purl.org/dc/terms/&gt; </span><span class="se">\n</span><span class="s1">PREFIX rdfs: &lt;http://www.w3.org/2000/01/rdf-schema#&gt;</span><span class="se">\n\n</span><span class="s1">SELECT ?idvalue ?phenofeatid ?phenofeatlabel</span><span class="se">\n</span><span class="s1">WHERE {</span><span class="se">\n</span><span class="s1">    # Find relevant phenopackets</span><span class="se">\n</span><span class="s1">    ?id a obo:IAO_0020000 ;</span><span class="se">\n</span><span class="s1">        sio:SIO_000300 ?idvalue .</span><span class="se">\n</span><span class="s1">    VALUES ?idvalue { &quot;PMID_26467218_individual_6_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_11_Kraoua_et_al__23&quot; &quot;PMID_26467218_individual_3_Denayer_et_al__22&quot; &quot;PMID_26467218_individual_9_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_4_Denayer_et_al__22&quot; &quot;PMID_26467218_individual_12_Present_study&quot; &quot;PMID_26467218_individual_1_De_Filippi_et_al__20&quot; &quot;PMID_26467218_individual_2_Runtuwene_et_al__21&quot; &quot;PMID_26467218_individual_9_M_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_5_Denayer_et_al__22&quot; &quot;PMID_26467218_individual_12_F_Present_study&quot; &quot;PMID_26467218_individual_8_Cirstea_et_al__14&quot; &quot;PMID_26467218_individual_10_Kraoua_et_al__23&quot; &quot;PMID_26467218_individual_7_Cirstea_et_al__14&quot; } .</span><span class="se">\n</span><span class="s1">    </span><span class="se">\n</span><span class="s1">    # Find phenotypic features</span><span class="se">\n</span><span class="s1">    ?id sio:SIO_000020 ?role .</span><span class="se">\n</span><span class="s1">    ?phenopacket sio:SIO_000228 ?role ;</span><span class="se">\n</span><span class="s1">                 sio:SIO_001403 ?phenofeat .</span><span class="se">\n</span><span class="s1">    ?phenofeatval sio:SIO_000628 ?phenofeat ;</span><span class="se">\n</span><span class="s1">                  a obo:NCIT_C16977 .</span><span class="se">\n</span><span class="s1">    ?phenofeat dcterms:identifier ?phenofeatid ;</span><span class="se">\n</span><span class="s1">               rdfs:label ?phenofeatlabel .</span><span class="se">\n</span><span class="s1">    </span><span class="se">\n</span><span class="s1">}&#39;</span>
</pre></div>
</div>
<p>A total of 326  phenotypes have been found. A sample of these results are shown below:</p>
<div class="output text_html"><div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>phenopacket ID</th>
      <th>phenotype ID</th>
      <th>phenotype label</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>94</th>
      <td>PMID_26467218_individual_9_Cirstea_et_al__14</td>
      <td>HP:0000028</td>
      <td>Cryptorchidism</td>
    </tr>
    <tr>
      <th>133</th>
      <td>PMID_26467218_individual_12_Present_study</td>
      <td>HP:0000767</td>
      <td>Pectus excavatum</td>
    </tr>
    <tr>
      <th>179</th>
      <td>PMID_26467218_individual_2_Runtuwene_et_al__21</td>
      <td>HP:0000256</td>
      <td>Macrocephaly</td>
    </tr>
    <tr>
      <th>184</th>
      <td>PMID_26467218_individual_9_M_Cirstea_et_al__14</td>
      <td>HP:0000508</td>
      <td>Ptosis</td>
    </tr>
    <tr>
      <th>198</th>
      <td>PMID_26467218_individual_9_M_Cirstea_et_al__14</td>
      <td>HP:0006699</td>
      <td>Premature atrial contractions</td>
    </tr>
    <tr>
      <th>207</th>
      <td>PMID_26467218_individual_5_Denayer_et_al__22</td>
      <td>HP:0000256</td>
      <td>Macrocephaly</td>
    </tr>
    <tr>
      <th>233</th>
      <td>PMID_26467218_individual_12_F_Present_study</td>
      <td>HP:0001561</td>
      <td>Polyhydramnios</td>
    </tr>
    <tr>
      <th>292</th>
      <td>PMID_26467218_individual_10_Kraoua_et_al__23</td>
      <td>HP:0001680</td>
      <td>Coarctation of aorta</td>
    </tr>
    <tr>
      <th>324</th>
      <td>PMID_26467218_individual_7_Cirstea_et_al__14</td>
      <td>HP:0006699</td>
      <td>Premature atrial contractions</td>
    </tr>
    <tr>
      <th>325</th>
      <td>PMID_26467218_individual_7_Cirstea_et_al__14</td>
      <td>HP:0002212</td>
      <td>Curly hair</td>
    </tr>
  </tbody>
</table>
</div></div><p>Given these results, the occurrence of each phenotype is calculated:</p>
<div class="output text_html"><div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>phenotype ID</th>
      <th>phenotype label</th>
      <th>counts</th>
      <th>occurrence percentage</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>HP:0000369</td>
      <td>Low-set ears</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>1</th>
      <td>HP:0000316</td>
      <td>Hypertelorism</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>2</th>
      <td>HP:0000957</td>
      <td>Cafe-au-lait spot</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>3</th>
      <td>HP:0012209</td>
      <td>Juvenile myelomonocytic leukemia</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>4</th>
      <td>HP:0000494</td>
      <td>Downslanted palpebral fissures</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>5</th>
      <td>HP:0004322</td>
      <td>Short stature</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>6</th>
      <td>HP:0000348</td>
      <td>High forehead</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>7</th>
      <td>HP:0000508</td>
      <td>Ptosis</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>8</th>
      <td>HP:0000358</td>
      <td>Posteriorly rotated ears</td>
      <td>14</td>
      <td>100.0%</td>
    </tr>
    <tr>
      <th>9</th>
      <td>HP:0000767</td>
      <td>Pectus excavatum</td>
      <td>13</td>
      <td>92.9%</td>
    </tr>
    <tr>
      <th>10</th>
      <td>HP:0001655</td>
      <td>Patent foramen ovale</td>
      <td>13</td>
      <td>92.9%</td>
    </tr>
    <tr>
      <th>11</th>
      <td>HP:0000256</td>
      <td>Macrocephaly</td>
      <td>13</td>
      <td>92.9%</td>
    </tr>
    <tr>
      <th>12</th>
      <td>HP:0001680</td>
      <td>Coarctation of aorta</td>
      <td>13</td>
      <td>92.9%</td>
    </tr>
    <tr>
      <th>13</th>
      <td>HP:0000483</td>
      <td>Astigmatism</td>
      <td>12</td>
      <td>85.7%</td>
    </tr>
    <tr>
      <th>14</th>
      <td>HP:0001270</td>
      <td>Motor delay</td>
      <td>12</td>
      <td>85.7%</td>
    </tr>
    <tr>
      <th>15</th>
      <td>HP:0000486</td>
      <td>Strabismus</td>
      <td>12</td>
      <td>85.7%</td>
    </tr>
    <tr>
      <th>16</th>
      <td>HP:0000465</td>
      <td>Webbed neck</td>
      <td>12</td>
      <td>85.7%</td>
    </tr>
    <tr>
      <th>17</th>
      <td>HP:0000545</td>
      <td>Myopia</td>
      <td>11</td>
      <td>78.6%</td>
    </tr>
    <tr>
      <th>18</th>
      <td>HP:0000563</td>
      <td>Keratoconus</td>
      <td>11</td>
      <td>78.6%</td>
    </tr>
    <tr>
      <th>19</th>
      <td>HP:0006699</td>
      <td>Premature atrial contractions</td>
      <td>11</td>
      <td>78.6%</td>
    </tr>
    <tr>
      <th>20</th>
      <td>HP:0002212</td>
      <td>Curly hair</td>
      <td>11</td>
      <td>78.6%</td>
    </tr>
    <tr>
      <th>21</th>
      <td>HP:0000978</td>
      <td>Bruising susceptibility</td>
      <td>11</td>
      <td>78.6%</td>
    </tr>
    <tr>
      <th>22</th>
      <td>HP:0001561</td>
      <td>Polyhydramnios</td>
      <td>10</td>
      <td>71.4%</td>
    </tr>
    <tr>
      <th>23</th>
      <td>HP:0001195</td>
      <td>Single umbilical artery</td>
      <td>10</td>
      <td>71.4%</td>
    </tr>
    <tr>
      <th>24</th>
      <td>HP:0032152</td>
      <td>Keratosis pilaris</td>
      <td>9</td>
      <td>64.3%</td>
    </tr>
    <tr>
      <th>25</th>
      <td>HP:0000028</td>
      <td>Cryptorchidism</td>
      <td>9</td>
      <td>64.3%</td>
    </tr>
    <tr>
      <th>26</th>
      <td>HP:0001639</td>
      <td>Hypertrophic cardiomyopathy</td>
      <td>3</td>
      <td>21.4%</td>
    </tr>
    <tr>
      <th>27</th>
      <td>HP:0001642</td>
      <td>Pulmonic stenosis</td>
      <td>3</td>
      <td>21.4%</td>
    </tr>
    <tr>
      <th>28</th>
      <td>HP:0001631</td>
      <td>Atrial septal defect</td>
      <td>1</td>
      <td>7.1%</td>
    </tr>
  </tbody>
</table>
</div></div></section>
<section id="question-3-matching-phenotypes">
<h3>Question 3: Matching Phenotypes<a class="headerlink" href="#question-3-matching-phenotypes" title="Link to this heading">#</a></h3>
<p>Do the found phenotypes overlap with the phenotypes observed in the undiagnosed child?</p>
<p>Multiple phenotypes have been observed in the undiagnosed child being broad neck, low-set ears and motor delay. In order to match the observed phenotypes with the phenotypes observed in the phenopackets, the <a class="reference external" href="https://hpo.jax.org/">Human Phenotype Ontology (HPO)</a> identifiers need to be retrieved. These identifiers are acquired by querying over this HPO ontology matching the phenotypes with the labels of the terms that are part of HPO. Below, a SPARQL query is shown that searches for the identifier for the broad neck phenotype:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="s2">&quot;PREFIX rdfs: &lt;http://www.w3.org/2000/01/rdf-schema#&gt;</span><span class="se">\n</span><span class="s2">PREFIX obo: &lt;http://purl.obolibrary.org/obo/&gt;</span><span class="se">\n\n</span><span class="s2">SELECT ?iri</span><span class="se">\n</span><span class="s2">WHERE {</span><span class="se">\n</span><span class="s2">    ?iri rdfs:label &#39;Broad neck&#39; .</span><span class="se">\n</span><span class="s2">}&quot;</span>
</pre></div>
</div>
<p>Executing this SPARQL query for all observed phenotypes results in the following identifiers:</p>
<div class="output text_html"><div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>id</th>
      <th>label</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>HP:0000475</td>
      <td>Broad neck</td>
    </tr>
    <tr>
      <th>1</th>
      <td>HP:0000369</td>
      <td>Low-set ears</td>
    </tr>
    <tr>
      <th>2</th>
      <td>HP:0001270</td>
      <td>Motor delay</td>
    </tr>
  </tbody>
</table>
</div></div><p>Next, the list of observed phenotypes can be compared with the list of phenotypes associated with the selection of phenopackets coming to the conclusion that the observed phenotypes Motor delay (HP:0001270), Low-set ears (HP:0000369)  are also found in the individuals represented in the phenopackets.</p>
<p>It is still possible to gather more information about the observed phenotype that has not been found in at least one of the phenopackets. One step would be to investigate the closeness of the observed phenotype to the phenotypes associated with Noonan syndrome 6 given their relative positions in the HPO ontology. To be more explicit, this closeness can be calculated by acquiring the shortest path length between phenotypes in a network. This network is built by adding each phenotype as a node. An edge between two nodes represents the relation of one phenotype being the subclass or superclass of another phenotype.</p>
<p>Given the path length distances between each pair of phenotypes, the phenotypes can be shown on a two-dimensional plot by applying multidimensional scaling. In this way, a clear overview is generated of all phenotypes and their similarities expressed in their positions in the two dimensional space. This plot is shown below:</p>
<p><img alt="image" src="_images/nras_matching_phenotypes.png" /></p>
<p>The phenotypes labeled in bold text are the phenotypes observed in the undiagnosed child. Again, it can be seen that the phenotypes low-set ears and motor delay are also found in patients with Noonan syndrome 6 given the colors of the points that represent the phenotypes. Interestingly, this plot also shows that the remaining observed phenotype “broad neck” seems to be similar to phenotype “webbed neck” that is also associated in group of individuals with this same diagnosis.</p>
</section>
</section>
<hr class="docutils" />
<section id="conclusion">
<h2>Conclusion<a class="headerlink" href="#conclusion" title="Link to this heading">#</a></h2>
<p>Given the mutations in the NRAS gene identified in individuals with Noonan syndrome 6 and the overlap and similarities between the observed phenotypes and the phenotypes associated with the individuals diagnosed with Noonan syndrome 6, there is reason to further investigate Noonan syndrome 6 as being a likely diagnosis for the child.</p>
<hr class="docutils" />
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="scenario4.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title">Scenario 4: Genetic Analysis on AML</p>
      </div>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#an-undiagnosed-child-with-delayed-motor-development-and-generalized-dysmorphism">An Undiagnosed Child with Delayed Motor Development and Generalized Dysmorphism</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#research-questions">Research Questions</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#question-1-gene-mutation-prevalence">Question 1: Gene Mutation Prevalence</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#question-2-phenotype-and-disease-prevalence">Question 2: Phenotype and Disease Prevalence</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#question-3-matching-phenotypes">Question 3: Matching Phenotypes</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#conclusion">Conclusion</a></li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Andra Waagmeester, Nuria Queralt Rosinach, Wolmar Nyberg, and Marco Roos
</p>

  </div>
  
  <div class="footer-item">
    

  <p class="copyright">
    
      © Copyright 2023.
      <br/>
    
  </p>

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>